docs/contributing/index.md

```diff
@@ -53,35 +79,167 @@ Tap, click, or hover over the GitHub Octocat icon for options.
 :alt: GitHub navigation menu 
 ```
 
-You can use this menu to quickly navigate to the source repository, open an issue, or suggest an edit to the current document.
-Of course, you can use whichever tools you like.
+You can use this menu to quickly navigate to the `documentation` source repository, open an issue, or suggest an edit to the current document.
+Note that this navigation convenience is provided only for the `documentation` repository.
 
-Next edit files, commit your changes, push them to the remote repository, and submit a pull request to resolve the issue.
 
-Members who subscribe to the repository will receive a notification and review your request.
+(contributing-quick-edits-label)=
 
+### Quick edits
 
-(contributing-roles-label)=
+Quick edits for minor issues, such as typographical errors, misspellings, and English grammar and syntax, can be made through the GitHub user interface.
 
-## Contributor Roles
+1.  Navigate to the repository as noted in {ref}`contributing-manage-on-github-label`.
+1.  Navigate with the `docs` directory to find the source file to edit.
+1.  Click the {guilabel}`pencil icon` to edit the file in the browser.
 
-Contributors to the Plone Documentation may perform one or many roles.
+    ```{image} /_static/edit-pencil-icon.png
+    :alt: GitHub Edit this file
+    ```
+1.  Make edits, add a commit message, select {guilabel}`Create a **new branch** for this commit and start a pull request`, then click {guilabel}`Propose changes`.
+1.  Make your pull request against the branch `6-dev`.
+1.  Members who subscribe to the repository will receive a notification and review your request.
 
-- **Plone users and developers** use this documentation because it is accurate and actively maintained.
-  People in these roles typically contribute minor corrections.
-  They should read {doc}`setup-build` and {doc}`writing-docs-guide`.
-- **Authors** create Plone Documentation.
-  They should read {doc}`setup-build` and {doc}`writing-docs-guide`.
-  They should also read {doc}`authors` for guidance and tips for writing good technical documentation.
 
+(contributing-large-edits-label)=
 
-(contributing-quality-requirements-label)=
+### Large edits
 
-## Documentation quality requirements
+For large edits, first follow the instructions in {doc}`setup-build`.
+
+Once you have your environment set up, then you can follow the standard practice for making a pull request.
+This practice differs depending on whether you are making contributions to only the core `documentation` files or `volto` files as well.
+
+
+(contributing-documentation-only-label)=
+
+### Working with only the `documentation` repository
+
+This section describes how to make contributions to files in the `documentation` repository only, and excludes `volto` files.
+
+1.  From the project root directory, synch your local `6-dev` branch with its remote.
```

Comment:
```suggestion
1.  From the project root directory, sync your local `6-dev` branch with its remote.
```

Reply:
I am showing my age. Trends over the last two decades show that "sync" is now the more common usage. Reference: https://grammarhow.com/sync-vs-synch/